brainmap-cms/static/brainmap-cms/img/org_logos/sgci-logo-sans-text.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 226.1 84" xml:space="preserve"><style>.stLogo{fill:#fff}</style><path class="stLogo" d="M64.3 29.4c-.9-1.1-2.1-1.9-3.6-2.5-1.5-.5-2.9-.8-4.3-.8-.8 0-1.6.1-2.4.3-.8.2-1.6.5-2.4.9-.7.4-1.4.9-1.8 1.6-.5.7-.7 1.4-.7 2.4 0 1.5.6 2.6 1.7 3.4 1.1.8 2.5 1.5 4.2 2 1.7.6 3.5 1.1 5.5 1.7 2 .6 3.8 1.4 5.5 2.4 1.7 1 3.1 2.4 4.2 4.1 1.1 1.7 1.7 4 1.7 6.9 0 2.8-.5 5.2-1.5 7.2-1 2.1-2.4 3.8-4.2 5.1-1.7 1.4-3.8 2.4-6.1 3-2.3.7-4.7 1-7.3 1-3.2 0-6.2-.5-9-1.5s-5.3-2.6-7.7-4.8l7.8-8.5c1.1 1.5 2.5 2.6 4.2 3.5 1.7.8 3.5 1.2 5.3 1.2.9 0 1.8-.1 2.7-.3.9-.2 1.7-.5 2.4-.9s1.3-1 1.7-1.6c.4-.7.7-1.4.7-2.3 0-1.5-.6-2.7-1.7-3.5-1.1-.9-2.6-1.6-4.3-2.2-1.7-.6-3.6-1.2-5.6-1.8-2-.6-3.9-1.4-5.6-2.4-1.7-1-3.2-2.4-4.3-4.1-1.1-1.7-1.7-3.9-1.7-6.6 0-2.7.5-5 1.6-7 1-2 2.4-3.7 4.2-5 1.7-1.4 3.8-2.4 6-3 2.3-.7 4.6-1 7.1-1 2.8 0 5.5.4 8.1 1.2 2.6.8 5 2.1 7.1 4l-7.5 7.9zM161.1 30.1c-1.1-1.3-2.4-2.3-3.9-3-1.6-.7-3.4-1-5.4-1-2.1 0-4 .4-5.7 1.2-1.7.8-3.3 1.9-4.5 3.3-1.3 1.4-2.3 3.1-3 5.1-.7 2-1.1 4.1-1.1 6.4 0 2.3.4 4.5 1.1 6.4.7 1.9 1.7 3.6 3 5s2.7 2.5 4.4 3.3c1.7.8 3.5 1.2 5.5 1.2 2.2 0 4.2-.5 5.9-1.4 1.7-.9 3.1-2.2 4.3-3.9l9.1 6.8c-2.1 2.9-4.8 5.1-8 6.5-3.2 1.4-6.5 2.1-9.9 2.1-3.9 0-7.4-.6-10.7-1.8-3.3-1.2-6.1-2.9-8.5-5.2-2.4-2.3-4.2-5-5.6-8.2-1.3-3.2-2-6.8-2-10.8 0-4 .7-7.6 2-10.8 1.3-3.2 3.2-6 5.6-8.2 2.4-2.3 5.2-4 8.5-5.2 3.3-1.2 6.8-1.8 10.7-1.8 1.4 0 2.9.1 4.4.4 1.5.3 3 .7 4.5 1.2 1.5.6 2.9 1.3 4.2 2.2 1.4.9 2.6 2.1 3.6 3.4l-8.5 6.8zM111.5 55.5c-1 .6-2.4 1.1-4.1 1.6-1.7.5-3.8.8-6.2.8-2.3 0-4.4-.4-6.3-1.2-1.9-.7-3.6-1.8-4.9-3.3-1.4-1.4-2.4-3.1-3.1-5s-1.1-4.1-1.1-6.4c0-2.3.4-4.4 1.1-6.4s1.8-3.6 3.1-5.1c1.4-1.4 3-2.5 4.9-3.3 1.9-.8 4-1.2 6.3-1.2 2.8 0 5.1.4 7 1.2 1.9.8 3.5 2 5 3.5l7.7-8.4c-2.5-2.3-5.4-4-8.8-4.9-3.3-1-7-1.4-10.9-1.4-3.9 0-7.4.6-10.7 1.8-3.3 1.2-6.1 3-8.5 5.2-2.4 2.3-4.2 5-5.6 8.2-1.3 3.2-2 6.8-2 10.8 0 4 .7 7.6 2 10.8 1.3 3.2 3.2 6 5.6 8.2 2.4 2.3 5.2 4 8.5 5.2 3.3 1.2 6.8 1.8 10.7 1.8 3.6 0 7.2-.4 10.7-1.1 3.5-.7 6.9-2 10.1-3.6V37.6l-10.5 5.2v12.7zM175.2 17h12.3v50h-12.3zM226.1 84V0l-20.8 10.4v63.2zM0 84V0l20.7 10.4v63.2z"></path></svg>
```

Comment:
__New SGCI Logo  File__
I opted to retain the image with text, and add an SVG without text because SVG's size better and are often smaller files.